src/entity.js

```diff
@@ -417,7 +417,7 @@ Entity.configure = function(options) {
     var keys = _.keys(mapping).sort();
     sign = function(properties) {
       var hash  = crypto.createHmac('sha512', this.__signingKey);
-      var buf   = new Buffer(4);
+      var buf   = Buffer.alloc(4);
```

Comment:
Nice!  I think that switching to `.alloc` and `.from` are good changes to make since we're already touching the library.  Since these changes are pretty easy to understand, I don't see them as being risky.